src/test/java/de/retest/recheck/cli/util/ReportCreator.java

```diff
@@ -46,19 +47,24 @@ public static String createReportFileWithoutDiffs( final TemporaryFolder folder
 
 	public static String createReportFileWithDiffs( final TemporaryFolder folder ) throws IOException {
 		final File result = folder.newFile( REPORT_WITH_DIFFS_FILE_NAME );
+		final File recheckFolder = folder.newFolder( "check.recheck" );
```

Comment:
`.recheck` can be either a folder or a ZIP file. Does it make sense to consider the persistence here?